### PR TITLE
EN language update / fix for m % 5 == 0 case

### PR DESCRIPTION
**Pull request by iainfm**
Hi,

Thanks for making yesterday's changes so quickly! I have changed the English slightly (we don't say 'IT JUST WAS' with time...not that I can recall anyway). It's more usual to say 'IT IS JUST AFTER'.

The 'before' text was appearing when (m % 5 == 0), so I've added a check for this to fix it.

Thanks again :)

### CODE REVIEW

File: wc_langPacks/wc_langPack_EN.js [hunk 1]
```diff
@@ -45,9 +45,9 @@ wc_addLanguagePack({
     // 5+3, 5+4 => nearly
     if (
       (settings.fuzzyTime == 'both' || settings.fuzzyTime == 'after') &&
-      m % 5 <= 2
+      m % 5 <= 2 && m % 5 != 0
     ) {
-      ret += 'JUST WAS ';
+      ret += 'IS JUST AFTER ';
```
Comment (Snapstromegon) [inline wc_langPacks/wc_langPack_EN.js:50]: Can you move the IS out to line 26?
Then you can remove the else.
Thanks for catching the 0 minutes bug.

File: wc_langPacks/wc_langPack_EN.js [hunk 2]
```diff
@@ -103,4 +103,4 @@ wc_addLanguagePack({
     }
     return ret;
   },
-});
+});
```
Comment (Snapstromegon) [inline wc_langPacks/wc_langPack_EN.js:106]: It's good practice to always keep a trailing empty line.
```suggestion
});

```

File: wc_langPacks/wc_langPack_EN.js [hunk 3]
```diff
@@ -45,9 +45,9 @@ wc_addLanguagePack({
...
     if (
       (settings.fuzzyTime == 'both' || settings.fuzzyTime == 'after') &&
-      m % 5 <= 2
+      m % 5 <= 2 && m % 5 != 0
     ) {
-      ret += 'JUST WAS ';
+      ret += 'IS JUST AFTER ';
     } else {
       ret += 'IS ';
     }
```
Comment (Snapstromegon) [inline wc_langPacks/wc_langPack_EN.js:53]: Can you move the IS out to line 26?
Then you can remove the else.
Thanks for catching the 0 minutes bug.

**Review by Snapstromegon — CHANGES_REQUESTED** [review]

File: wc_langPacks/wc_langPack_EN.js (hunk 3 above)
Comment (iainfm) [reply wc_langPacks/wc_langPack_EN.js:53]: Yes, great idea. Much more efficient :) I've done this to my local copy and it works fine. Do I need to create a new PR for this?

Comment (Snapstromegon) [reply wc_langPacks/wc_langPack_EN.js:53]: You just push to your PR branch - this automatically updates the PR.

Comment (iainfm) [reply wc_langPacks/wc_langPack_EN.js]: I *think* I have done this :)

**Review by Snapstromegon — APPROVED** [review]
Thanks, looks good to me

**Pull request merged**